src/routes.ts

```diff
@@ -0,0 +1,11 @@
+import { Router } from 'express'
+
+const routes = Router();
+
+routes.get("/", (req, res) => {
```

Comment:
For readability, we will use full, self-describing variable names.

`req` -> `request`
`res` -> `response`